test/Microsoft.NET.Sdk.BlazorWebAssembly.Tests/BlazorLegacyIntegrationTest60.cs

```diff
@@ -44,7 +44,7 @@ public void Build60Hosted_Works()
             new FileInfo(Path.Combine(serverBuildOutputDirectory, $"{testAsset}.Shared.dll")).Should().Exist();
         }
 
-        [PlatformSpecificFact(TestPlatforms.Windows)]
+        [RequiresMSBuildVersionFact("17.12", Reason = "Needs System.Text.Json 8.0.5")]
```

Comment:
This change didn't work as the test now runs on non Windows platforms:

> Test Microsoft.NET.Sdk.BlazorWebAssembly.Tests.BlazorLegacyIntegrationTest60.Publish60Hosted_Works has failed.

@marcpopMSFT how do I use both the `RequiresMSBuildVersionFact` and `PlatformSpecificFact` logic in this repo? I would have assumed that the `SkipOnPlatform` attribute below makes sure that this doesn't run on linux or osx but apparently it does.

Reply:
Going by https://github.com/dotnet/sdk/blob/93788ad6cc8841e60e60a3ea780ed0a14f85ac48/test/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishAnAotApp.cs#L438-L442 we can wrap the body in condition?

Reply:
am11 is correct, we don't have a single Fact/Theory that limits both the msbuild version and the platform so you have to use an alternative check. So this test wasn't meant to be running then and that's the fix.

Reply:
Ah there is a better one:
```suggestion
        [WindowsOnlyRequiresMSBuildVersionFact("17.12", Reason = "Needs System.Text.Json 8.0.5")]
```

Reply:
forgot we had that one, good catch.